pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -91,6 +91,10 @@ type ACMEChallengeSolver struct {
 
 	// +optional
 	DNS01 *ACMEChallengeSolverDNS01 `json:"dns01,omitempty"`
+
+	// Controls how the self check behaves upon failure.
+	// +optional
+	FailurePolicy cmmeta.ACMESelfCheckFailurePolicy `json:"failurePolicy"`
```

Comment:
``` 
➜ kubectl apply -f pebble.yaml                       
The ClusterIssuer "pebble" is invalid: spec.acme.solvers.failurePolicy: Unsupported value: "": supported values: "WaitForever", "Ignore"
```

I think this is related to what @munnerz said the other day. Making this field a pointer probably solves it. This is probably a good first step, even though @munnerz mentioned a overhaul of how defualt values work by using some kubebuilder annotation.

Reply:
As James mentioned, `omitempty` solved it!